appveyor.yml

```diff
@@ -1,250 +0,0 @@
-      PYTHON_VERSION: 3.9
-      PYTHON_ARCH: 32
-      TEST_PART: Libraries
-
-    - PYTHON: C:\Python39
-      PYTHON_VERSION: 3.9
-      PYTHON_ARCH: 32
-      TEST_PART: Base
-
-    - PYTHON: C:\Python38-x64
-      PYTHON_VERSION: 3.8
-      PYTHON_ARCH: 64
-      TEST_PART: Libraries
-
-    - PYTHON: C:\Python38-x64
-      PYTHON_VERSION: 3.8
-      PYTHON_ARCH: 64
-      TEST_PART: Base
-
-    - PYTHON: C:\Python38
-      PYTHON_VERSION: 3.8
-      PYTHON_ARCH: 32
-      TEST_PART: Libraries
-
-    - PYTHON: C:\Python38
-      PYTHON_VERSION: 3.8
-      PYTHON_ARCH: 32
-      TEST_PART: Base
-
-    - PYTHON: C:\Python37-x64
-      PYTHON_VERSION: 3.7
-      PYTHON_ARCH: 64
-      TEST_PART: Libraries
-
-    - PYTHON: C:\Python37-x64
-      PYTHON_VERSION: 3.7
-      PYTHON_ARCH: 64
-      TEST_PART: Base
-
-    - PYTHON: C:\Python37
-      PYTHON_VERSION: 3.7
-      PYTHON_ARCH: 32
-      TEST_PART: Libraries
-
-    - PYTHON: C:\Python37
-      PYTHON_VERSION: 3.7
-      PYTHON_ARCH: 32
-      TEST_PART: Base
-
-    - PYTHON: C:\Python36
-      PYTHON_VERSION: 3.6
-      PYTHON_ARCH: 32
-      TEST_PART: Libraries
-
-    - PYTHON: C:\Python36
-      PYTHON_VERSION: 3.6
-      PYTHON_ARCH: 32
-      TEST_PART: Base
-
-init:
-  - ECHO %PYTHON% %PYTHON_VERSION% %PYTHON_ARCH%
-  - ECHO \"%APPVEYOR_SCHEDULED_BUILD%\"
-  # If there is a newer build queued for the same PR, cancel this one.
-  # The AppVeyor 'rollout builds' option is supposed to serve the same
-  # purpose but it is problematic because it tends to cancel builds pushed
-  # directly to master instead of just PR builds (or the converse).
-  # credits: JuliaLang developers.
-  - ps: |
-      if ($env:ENABLE_RDP_ACCESS -eq "init" ) {
-          $blockRdp = $true;
-          iex ((new-object net.webclient).DownloadString(
-          'https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1')) }
-  - ps: if ($env:APPVEYOR_PULL_REQUEST_NUMBER -and $env:APPVEYOR_BUILD_NUMBER -ne ((Invoke-RestMethod `
-        https://ci.appveyor.com/api/projects/$env:APPVEYOR_ACCOUNT_NAME/$env:APPVEYOR_PROJECT_SLUG/history?recordsNumber=50).builds | `
-        Where-Object pullRequestId -eq $env:APPVEYOR_PULL_REQUEST_NUMBER)[0].buildNumber) { `
-          throw "There are newer queued builds for this pull request, failing early." }
-  # Free up some disk space; exceeding the limit of 90 GB results in a
-  # locked-out account. The Qt directory takes over 30 GB. Do this in the
-  # background.
-  - ps: Start-Process -FilePath "$env:comspec" -ArgumentList "/c rmdir C:\Qt /s /q"
-
-
-cache:
-  # Cache downloaded pip packages and built wheels.
-  - '%LOCALAPPDATA%\pip\Cache\http'
-  - '%LOCALAPPDATA%\pip\Cache\wheels'
-
-
-install:
-  # Enable long paths (needs reboot)
-  - ps: Set-ItemProperty -Path 'HKLM:\SYSTEM\CurrentControlSet\Control\FileSystem' -Name 'LongPathsEnabled' -Value 1
-  - ps: Start-Sleep -s 10
-  - ps: Restart-Computer -Force
-  - ps: Start-Sleep -s 10
```

Comment:
Hmm, wonder if we still need to explicitly enable long paths on Github actions...

Reply:
[I see](https://github.com/pyinstaller/pyinstaller/runs/2271880377?check_suite_focus=true#step:12:973) this is no longer a problem.